android/app/src/main/kotlin/com/yubico/authenticator/fido/FidoManager.kt

```diff
@@ -603,6 +611,44 @@ class FidoManager(
             ).toString()
         }
 
+    private suspend fun enableEnterpriseAttestation(): String =
+        connectionHelper.useSession(FidoActionDescription.EnableEnterpriseAttestation) { fidoSession ->
+
+            val uvAuthProtocol = getPreferredPinUvAuthProtocol(fidoSession.cachedInfo)
+            val clientPin = ClientPin(fidoSession, uvAuthProtocol)
+            val token = if (pinStore.hasPin()) {
+                clientPin.getPinToken(
+                    pinStore.getPin(),
+                    getPinPermissionsACFG(fidoSession),
+                    null
+                )
+            } else null
+            val config = Config(fidoSession, uvAuthProtocol, token)
```

Comment:
This line will fail if Config.isSupported() returns false, throwing an exception. Since this can only work when it is supported, the `getPinPermissionsACFG` method can just be replaced with `ClientPin.PIN_PERMISSON_ACFG`, can't it?

Reply:
yes, I am updating this

Reply:
Rewrote this part.